cea/interfaces/dashboard/dashboard.py

```diff
@@ -141,12 +141,10 @@ def join_paths(path_list, loop_index):
     # the protocol for the Connection messages is tuples ('stdout'|'stderr', str)
     app.workers = {}  # script-name -> (Process, Connection)
 
-    print("starting webbrowser timer")
-    threading.Timer(0.5, lambda: webbrowser.open('http://localhost:5050')).start()
-    print("started webbrowser timer")
     print("start socketio.run")
     socketio.run(app, host='localhost', port=5050)
     print("done socketio.run")
```

Comment:
Does the 'done' here mean `socketio.run` "has started running" or "has closed / stopped running", as it will not be printed until `socketio.run` ends since it's blocking

Reply:
`socketio.run` starts the Flask webserver. So yes, it's only printed when the application ends.